src/test/rgw/bucket_notification/README.rst

```diff
@@ -0,0 +1 @@
+TO BE FILLED
```

Comment:
Is this going to get filled yet? If not I'd just remove it for now.

Reply:
Ohh, thanks for pointing this out. I've to write to it.

Reply:
please add info on manual/local testing here

Reply:
Okay, sure